akka-actor-typed/src/main/scala/akka/actor/typed/internal/receptionist/ReceptionistMessages.scala

```diff
@@ -4,11 +4,12 @@
 
 package akka.actor.typed.internal.receptionist
 
+import java.util
```

Comment:
Should we be specific vs an open-ended util?

Reply:
Yeah, some weird autoimport from IntelliJ I think, I'll remove it